db/migrate/20190617160447_create_vehicles.rb

```diff
@@ -0,0 +1,13 @@
+class CreateVehicles < ActiveRecord::Migration[5.2]
+  def change
+    create_table :vehicles, id: :uuid do |t|
+      t.uuid :assessment_id
```

Comment:
I suggest using Rails' `belongs_to` keyword. 
```suggestion
      t.belongs_to :assessment, foreign_key: true, null: false, type: :uuid
```